ERROR 2011-04-20 16:25:10,327 [9] StudentAssessment.Data.TransactionData [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = QTEST1101
SOPTYPE = 1
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE
NONINVEN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
ITEMNMBR = TESTITEM


<taSopLineIvcInsert><SOPTYPE>1</SOPTYPE><SOPNUMBE>QTEST1101</SOPNUMBE><CUSTNMBR>JRPA</CUSTNMBR><DOCDATE>4/20/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>TESTITEM</ITEMNMBR><UNITPRCE>2500.00</UNITPRCE><XTNDPRCE>2500.00</XTNDPRCE><QUANTITY>0</QUANTITY><MRKDNPCT>0.00</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>Test Item</ITEMDESC><DOCID>STDQTE</DOCID><QUOTEQTYTOINV>1</QUOTEQTYTOINV><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 122 - saveToSOP 129 
ERROR 2011-04-20 16:25:10,454 [9] StudentAssessment.frmAssessment [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = QTEST1101
SOPTYPE = 1
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE
NONINVEN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
ITEMNMBR = TESTITEM


<taSopLineIvcInsert><SOPTYPE>1</SOPTYPE><SOPNUMBE>QTEST1101</SOPNUMBE><CUSTNMBR>JRPA</CUSTNMBR><DOCDATE>4/20/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>TESTITEM</ITEMNMBR><UNITPRCE>2500.00</UNITPRCE><XTNDPRCE>2500.00</XTNDPRCE><QUANTITY>0</QUANTITY><MRKDNPCT>0.00</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>Test Item</ITEMDESC><DOCID>STDQTE</DOCID><QUOTEQTYTOINV>1</QUOTEQTYTOINV><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 130
   at StudentAssessment.Data.TransactionData.SaveTransaction(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 136
   at StudentAssessment.Adapter.TransactionAdapter.SaveTransaction(Transaction transaction, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Adapter\TransactionAdapter.cs:line 39
   at StudentAssessment.frmAssessment.btnPrint_Click(Object sender, EventArgs e) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmAssessment.cs:line 482 - btnPrint_Click 499 
ERROR 2011-04-26 11:57:41,423 [9] StudentAssessment.frmReportViewer [(null)] - System.NullReferenceException: Object reference not set to an instance of an object.
   at StudentAssessment.frmReportViewer.initializeReport(rptAssessment report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 69 - initializeReport 74 
ERROR 2011-04-26 15:11:31,640 [8] StudentAssessment.Data.TransactionData [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 7227  Stored Procedure taSopMultiBin  Error Description = A sales order return bin is not set up for this item or site
Node Identifier Parameters: taSopMultiBin                                      
SOPNUMBE = STDINV2281
SOPTYPE = 4
ITEMNMBR = IDN-SISC-20--0002
Related Error Code Parameters for Node : taSopMultiBin                                      
BIN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
SOPTYPE = 4
ITEMNMBR = IDN-SISC-20--0002
LOCNCODE = WAREHOUSE


Error Number = 5996  Stored Procedure taSopLineIvcInsert  Error Description = Unable to update SOP Default Bins
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = STDINV2281
SOPTYPE = 4
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
AutoAssignBin =  Note: This parameter was not passed in, no value for the parameter will be returned. 


<taSopLineIvcInsert><SOPTYPE>4</SOPTYPE><SOPNUMBE>STDINV2281</SOPNUMBE><CUSTNMBR>AARONFIT0001</CUSTNMBR><DOCDATE>4/26/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>IDN-SISC-20--0002</ITEMNMBR><QUANTITY>3</QUANTITY><MRKDNPCT>0</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>Sociology</ITEMDESC><QTYINSVC>3</QTYINSVC><DOCID>RTN</DOCID><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 7227  Stored Procedure taSopMultiBin  Error Description = A sales order return bin is not set up for this item or site
Error Number = 5996  Stored Procedure taSopLineIvcInsert  Error Description = Unable to update SOP Default Bins

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 122 - saveToSOP 129 
ERROR 2011-04-26 15:11:31,875 [8] StudentAssessment.frmAssessment [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 7227  Stored Procedure taSopMultiBin  Error Description = A sales order return bin is not set up for this item or site
Node Identifier Parameters: taSopMultiBin                                      
SOPNUMBE = STDINV2281
SOPTYPE = 4
ITEMNMBR = IDN-SISC-20--0002
Related Error Code Parameters for Node : taSopMultiBin                                      
BIN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
SOPTYPE = 4
ITEMNMBR = IDN-SISC-20--0002
LOCNCODE = WAREHOUSE


Error Number = 5996  Stored Procedure taSopLineIvcInsert  Error Description = Unable to update SOP Default Bins
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = STDINV2281
SOPTYPE = 4
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
AutoAssignBin =  Note: This parameter was not passed in, no value for the parameter will be returned. 


<taSopLineIvcInsert><SOPTYPE>4</SOPTYPE><SOPNUMBE>STDINV2281</SOPNUMBE><CUSTNMBR>AARONFIT0001</CUSTNMBR><DOCDATE>4/26/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>IDN-SISC-20--0002</ITEMNMBR><QUANTITY>3</QUANTITY><MRKDNPCT>0</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>Sociology</ITEMDESC><QTYINSVC>3</QTYINSVC><DOCID>RTN</DOCID><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 7227  Stored Procedure taSopMultiBin  Error Description = A sales order return bin is not set up for this item or site
Error Number = 5996  Stored Procedure taSopLineIvcInsert  Error Description = Unable to update SOP Default Bins

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 130
   at StudentAssessment.Data.TransactionData.SaveTransaction(Transaction assessment, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Data\TransactionData.cs:line 136
   at StudentAssessment.Adapter.TransactionAdapter.SaveTransaction(Transaction transaction, String documentID, String defaultSiteID) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Adapter\TransactionAdapter.cs:line 39
   at StudentAssessment.frmAssessment.btnPrint_Click(Object sender, EventArgs e) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmAssessment.cs:line 483 - btnPrint_Click 526 
ERROR 2011-04-27 13:16:08,410 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:16:33,637 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:17:42,422 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:20:42,910 [10] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:21:43,145 [10] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:31:02,908 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Load report failed. ---> System.Runtime.InteropServices.COMException (0x80004005): The system cannot find the file specified.
   at CrystalDecisions.ReportAppServer.ClientDoc.ReportClientDocumentClass.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.Open(Object& DocumentPath, Int32 Options)
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   --- End of inner exception stack trace ---
   at CrystalDecisions.ReportAppServer.ReportClientDocumentWrapper.EnsureDocumentIsOpened()
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename, OpenReportMethod openMethod, Int16 parentJob)
   at CrystalDecisions.CrystalReports.Engine.ReportDocument.Load(String filename)
   at StudentAssessment.frmReportViewer.initializeReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 52 - initializeReport 57 
ERROR 2011-04-27 13:36:21,866 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:36:37,581 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:37:31,284 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:37:45,514 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:37:57,741 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:40:07,358 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:40:13,809 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:40:20,090 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:40:25,680 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:40:35,644 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:41:44,182 [10] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:50:58,272 [8] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 13:55:44,612 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:04:25,678 [8] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:04:51,454 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:08:55,389 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:09:23,894 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:10:03,229 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-04-27 14:10:52,177 [9] StudentAssessment.frmReportViewer [(null)] - System.Exception: Exception of type 'System.Exception' was thrown.
   at StudentAssessment.frmReportViewer.displayReport(ReportClass report, KeyValuePair`2[] reportParameters) in D:\DATA on D\SISC\project\Southville\Student_Assessment\Basic_Ed_Assessment\frmReportViewer.cs:line 53 - displayReport 57 
ERROR 2011-10-12 06:44:51,361 [11] StudentAssessment.Data.TransactionData [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 106  Stored Procedure taSopLineIvcInsert  Error Description = Location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = 000010
SOPTYPE = 3
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE


Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = 000010
SOPTYPE = 3
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE
NONINVEN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
ITEMNMBR = UNT-SISC-20--0004


<taSopLineIvcInsert><SOPTYPE>3</SOPTYPE><SOPNUMBE>000010</SOPNUMBE><CUSTNMBR>TEST</CUSTNMBR><DOCDATE>10/12/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>UNT-SISC-20--0004</ITEMNMBR><UNITPRCE>1260.00</UNITPRCE><XTNDPRCE>3780.00</XTNDPRCE><QUANTITY>3</QUANTITY><MRKDNPCT>0.00</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>GenEd Unit</ITEMDESC><DOCID>INV</DOCID><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 106  Stored Procedure taSopLineIvcInsert  Error Description = Location code does not exist in inventory
Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at System.Runtime.Remoting.Proxies.RealProxy.HandleReturnMessage(IMessage reqMsg, IMessage retMsg)
   at System.Runtime.Remoting.Proxies.RealProxy.PrivateInvoke(MessageData& msgData, Int32 type)
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in C:\Documents and Settings\Administrator\Desktop\working folder\Student_Assessment\Data\TransactionData.cs:line 122 - saveToSOP 129 
ERROR 2011-10-12 06:44:51,962 [11] StudentAssessment.frmAssessment [(null)] - Microsoft.Dynamics.GP.eConnect.eConnectException: Sql procedure error codes returned:
Error Number = 106  Stored Procedure taSopLineIvcInsert  Error Description = Location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = 000010
SOPTYPE = 3
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE


Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory
Node Identifier Parameters: taSopLineIvcInsert                                 
SOPNUMBE = 000010
SOPTYPE = 3
Related Error Code Parameters for Node : taSopLineIvcInsert                                 
LOCNCODE = WAREHOUSE
NONINVEN =  Note: This parameter was not passed in, no value for the parameter will be returned. 
ITEMNMBR = UNT-SISC-20--0004


<taSopLineIvcInsert><SOPTYPE>3</SOPTYPE><SOPNUMBE>000010</SOPNUMBE><CUSTNMBR>TEST</CUSTNMBR><DOCDATE>10/12/2011</DOCDATE><LOCNCODE>WAREHOUSE</LOCNCODE><ITEMNMBR>UNT-SISC-20--0004</ITEMNMBR><UNITPRCE>1260.00</UNITPRCE><XTNDPRCE>3780.00</XTNDPRCE><QUANTITY>3</QUANTITY><MRKDNPCT>0.00</MRKDNPCT><PRCLEVEL>RETAIL</PRCLEVEL><ITEMDESC>GenEd Unit</ITEMDESC><DOCID>INV</DOCID><UOFM>Unit</UOFM></taSopLineIvcInsert> ---> Microsoft.Dynamics.GP.eConnect.eConnectException: Error Number = 106  Stored Procedure taSopLineIvcInsert  Error Description = Location code does not exist in inventory
Error Number = 110  Stored Procedure taSopLineIvcInsert  Error Description = Item number/location code does not exist in inventory

   --- End of inner exception stack trace ---

Server stack trace: 
   at Microsoft.Dynamics.GP.eConnect.eConnectMethods.eConnect_EntryPoint(String ConnectionString, ConnectionStringType ConnectionType, String sXML, SchemaValidationType ValidationType, String eConnectSchema)
   at System.Runtime.Remoting.Messaging.StackBuilderSink._PrivateProcessMessage(IntPtr md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.PrivateProcessMessage(RuntimeMethodHandle md, Object[] args, Object server, Int32 methodPtr, Boolean fExecuteInContext, Object[]& outArgs)
   at System.Runtime.Remoting.Messaging.StackBuilderSink.SyncProcessMessage(IMessage msg, Int32 methodPtr, Boolean fExecuteInContext)

Exception rethrown at [0]: 
   at StudentAssessment.Data.TransactionData.saveToSOP(Transaction assessment, String documentID, String defaultSiteID) in C:\Documents and Settings\Administrator\Desktop\working folder\Student_Assessment\Data\TransactionData.cs:line 130
   at StudentAssessment.Data.TransactionData.SaveTransaction(Transaction assessment, String documentID, String defaultSiteID) in C:\Documents and Settings\Administrator\Desktop\working folder\Student_Assessment\Data\TransactionData.cs:line 136
   at StudentAssessment.Adapter.TransactionAdapter.SaveTransaction(Transaction transaction, String documentID, String defaultSiteID) in C:\Documents and Settings\Administrator\Desktop\working folder\Student_Assessment\Adapter\TransactionAdapter.cs:line 39
   at StudentAssessment.frmAssessment.btnPrint_Click(Object sender, EventArgs e) in C:\Documents and Settings\Administrator\Desktop\working folder\Student_Assessment\Basic_Ed_Assessment\frmAssessment.cs:line 493 - btnPrint_Click 536 
